### PR TITLE
Updates to TODO file and significant improvements to README.md

### PR DESCRIPTION
**Pull request by Jacalz**
A lot of great little fixes to the README and a small fix to TODO :+1:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,49 +1,65 @@
 SC Controller [![Build Status](https://travis-ci.org/kozec/sc-controller.svg?branch=master)](https://travis-ci.org/kozec/sc-controller)
 =============
 
-User-mode driver, mapper and GTK3 based GUI for Steam Controller, DS4 and similar controllers.
+User-mode driver, mapper and GTK3 based GUI for Steam Controller, DS4, Xbox 360 and similar controllers.
```
Comment (kozec) [inline README.md]: Just a small issue here, Xbox 360 controller is not actually supported. Even adding it as evdev device would probably end bad, as sc-controller would start picking inputs from itself.

File: README.md [hunk 2]
```diff
@@ -1,49 +1,65 @@
...
+### Arch, Manjaro (arch-based distros):
+Found in [AUR](https://aur.archlinux.org/packages/sc-controller-git/)
+
+### Solus:
+Search for `sc-controller` in Software Center or run `sudo eopkg it sc-controller` from a terminal.
+
+### Exherbo:
+Found in [hardware](https://git.exherbo.org/summer/packages/input/sc-controller)
+
+### Void Linux:
```
Comment (kozec) [inline README.md]: Larger headers are probably fine, but listing distros like this makes it much less readable.

File: README.md (hunk 1 above)
Comment (Jacalz) [reply README.md]: I got my Xbox360 controller to work pretty well, but I see to get this updated in short

File: README.md (hunk 2 above)
Comment (Jacalz) [reply README.md]: I do not quite agree with you here, but I will do as you say :+1:

Comment (Jacalz) [reply README.md]: Okay. I changed my mind, with some improvements it can look really good...

**Comment by Jacalz** [comment]
This is now ready for a re-review :slightly_smiling_face:

**Comment by Jacalz** [comment]
I don’t have anything planned right now, so you should be fine to merge 🙂 I will open a new PR if I figure something out 👍

**Comment by kozec** [comment]
Ok, thanks for your help :)

**Comment by Jacalz** [comment]
You’re welcome :)

**Pull request merged**